src/IB/ConstraintIBMethod.cpp

```diff
@@ -661,33 +651,19 @@ ConstraintIBMethod::putToDatabase(Pointer<Database> db)
     // Put the following quantities to restart database.
     for (int struct_no = 0; struct_no < d_no_structures; ++struct_no)
     {
-        std::ostringstream posidentifier;
-        posidentifier << "POSN_COM_STRUCT_" << struct_no;
-        db->putDoubleArray(posidentifier.str(), &d_center_of_mass_current[struct_no][0], 3);
+        db->putDoubleArray("POSN_COM_STRUCT_" + std::to_string(struct_no), &d_center_of_mass_current[struct_no][0], 3);
```

Comment:
It would be clearer here and below if we just had
```cpp
const std::string struct_str = std::to_string(struct_no);
```
instead of all the calls to `std::to_string` with the same argument.